AATAMS/AATAMS_acoustic/check_csv.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
```

Comment:
Feel free to take with a grain of salt... I'd probably expand on it to take more advantage of the set operations to report on what the differences actually were to save time digging and manually comparing, e.g.:

```python
#!/usr/bin/env python
"""
Check that csv file has required fields 
"""
import os
import pandas
import sys

EXPECTED_HEADERS = {'transmitter_id', 'installation_name', 'station_name', 'receiver_name', 'detection_timestamp',
                    'longitude', 'latitude', 'sensor_value', 'sensor_unit', 'FDA_QC', 'Velocity_QC', 'Distance_QC',
                    'DetectionDistribution_QC', 'DistanceRelease_QC', 'ReleaseDate_QC', 'ReleaseLocation_QC',
                    'Detection_QC'}


def main(csvfile):
    # CSV file to extract array of formatted data
    data = pandas.read_csv(csvfile, delimiter=';', header=0)
    actual_headers = set(data.columns.values)

    # compare expected with actual headers, do nothing if they are equal
    if actual_headers != EXPECTED_HEADERS:
        only_in_actual = list(actual_headers.difference(EXPECTED_HEADERS))
        only_in_expected = list(EXPECTED_HEADERS.difference(actual_headers))

        error_message = ["Columns in file '{csvfile}' don't match expected columns".format(csvfile=csvfile)]
        # print report on what the differences were between the expected headers and the actual headers
        if only_in_actual:
            error_message.append("Unexpected columns: {only_in_actual}".format(only_in_actual=only_in_actual))
        if only_in_expected:
            error_message.append("Missing columns: {only_in_expected}".format(only_in_expected=only_in_expected))

        sys.exit(os.linesep.join(error_message))


if __name__ == '__main__':
    try:
        input_file = sys.argv[1]
    except IndexError:
        sys.exit("usage: {script} CSVFILE".format(script=sys.argv[0]))
    main(input_file)
```
Just exits silently with success if they match, otherwise prints handy report on what the problem is, e.g.

```bash
/home/lwgordon/check_csv/bin/python /home/lwgordon/check_csv.py test.csv
Columns in file 'test.csv' don't match expected columns
Unexpected columns: ['dsadsa.2', 'dsadsad', 'dsadsa.1', 'dsadsa']
Missing columns: ['installation_name', 'transmitter_id', 'ReleaseLocation_QC', 'sensor_unit', 'ReleaseDate_QC', 'detection_timestamp', 'longitude', 'station_name', 'Detection_QC', 'sensor_value', 'DetectionDistribution_QC', 'Velocity_QC', 'latitude', 'receiver_name', 'FDA_QC', 'DistanceRelease_QC', 'Distance_QC']

Process finished with exit code 1
```